examples/cython_polar/setup.py

```diff
@@ -0,0 +1,18 @@
+from distutils.core import setup
+from distutils.extension import Extension
+from Cython.Build import cythonize
+
+codec = Extension(
+    name="codec",
+    sources=["codec.pyx"],
+    libraries=["aff3ct-2.3.5-384-gec40f26"],
```

Comment:
This is hardcoded, it will not work with future commits

Reply:
Ack. Working on letting python parse the version.

Reply:
Done.